File: java/console/src/main/java/com/lacunasoftware/signer/sample/scenarios/DeleteDocumentScenario.java
package com.lacunasoftware.signer.sample.scenarios;

import com.lacunasoftware.signer.documents.CreateDocumentResult;
import com.lacunasoftware.signer.javaclient.exceptions.RestException;

import java.io.IOException;
import java.util.UUID;

public class DeleteDocumentScenario extends Scenario {
    /**
     * This scenario demonstrates how to delete a document using it's ID.
     */
    @Override
    public void Run() throws IOException, RestException {

        //1 - You need a document id
        CreateDocumentResult document = createDocument();
        UUID docId = document.getDocumentId();

        //2 - Call the api method to delete the document and pass the document Id as parameter
        signerClient.deleteDocument(docId);
    }
}

File: nodejs/scenarios/deleteDocumentScenario.ts
import { DocumentsApi } from "signer-node-client";
import { config, CreateDocument } from "./scenario";

const documentsApi = new DocumentsApi(config)

CreateDocument().then((res) => {
    //1 - You need a document id
    const docId = res.documentId;

    //2 - Call the api method to delete the document and pass the document Id as parameter
    documentsApi.apiDocumentsIdDelete(docId).then((res) => {
        if(res.status === 200){
            console.log("Document", docId, "deleted");
        }
    })
});